spec/scss_converter_spec.rb

```diff
@@ -388,7 +390,6 @@ def converter(overrides = {})
 
       it "contains relevant sass sources" do
         sources = sourcemap_data["sources"]
-        expect(sources).to include("main.scss")
```

Comment:
So sourcemaps from sass-embedded doesn't reference primary load-point?

Reply:
This is because the main file in this test does only have two `@import`. If you add some css rules, it will be listed. That’s a small behavior change of dart-sass.